recordedit/recordEdit.app.js

```diff
@@ -318,6 +321,7 @@
                 }
             });
         } catch (exception) {
```

Comment:
I don't think so. That would force us to move the code for the run block into a function just so we can wrap it with a try/catch.